packages/sdk-core/src/main/kotlin/ly.iterative.itly/CallOptions.kt

```diff
@@ -0,0 +1,7 @@
+package ly.iterative.itly
+
+class CallOptions @JvmOverloads constructor(
```

Comment:
Can CallOptions directly extend Map<String, PluginCallOptions>? It might make it easier to work with. `callOptions?.get(plugin.id())` vs `callOptions?.pluginToOptions?.get(plugin.id())`